Lib/test/test_tracemalloc.py

```diff
@@ -1141,6 +1142,37 @@ def __del__(self, untrack=_testcapi.tracemalloc_untrack):
         """)
         assert_python_ok("-c", code)
 
+    def test_trace_refs(self):
+        def f():
+            l = []
+            del l
+
+        def g():
+            l = [], []
+            del l
+
+        tracemalloc.start()
+
+        try:
+            tracemalloc.clear_traces()
+            f()
+            refs = tracemalloc.get_traced_refs()
+            if refs == (1, 0):
+                warnings.warn("ceval Py_DECREF doesn't emit PyRefTracer_DESTROY in this build")
```

Comment:
I expect a test failure, not a warning. In which case we would fall into this case? And why ignoring this failure?